test/e2e/main_test.go

```diff
@@ -72,9 +73,15 @@ func TestE2E(tapi *testing.T) {
 	for _, node := range nodeList.Items {
 		nodes = append(nodes, node.Name)
 	}
-	knmstateReporter := knmstatereporter.New("test_logs/e2e", framework.Global.Namespace, nodes)
-	junitReporter := ginkgoreporters.NewJUnitReporter("junit.functest.xml")
-	RunSpecsWithDefaultAndCustomReporters(t, "E2E Test Suite", []Reporter{junitReporter, knmstateReporter})
+
+	reporters := make([]Reporter, 0)
+	reporters = append(reporters, knmstatereporter.New("test_logs/e2e", framework.Global.Namespace, nodes))
+	reporters = append(reporters, ginkgoreporters.NewJUnitReporter("junit.functest.xml"))
+	if qereporters.JunitOutput != "" {
+		reporters = append(reporters, qereporters.NewJunitReporter())
+	}
+
+	RunSpecsWithDefaultAndCustomReporters(t, "E2E Test Suite", reporters)
```

Comment:
Backport the fix

Reply:
@phoracek added a commit https://github.com/nmstate/kubernetes-nmstate/pull/453/commits/fa54c38ad342a549ee58c690778715ce698c6782 to address this.